drupal/patches/null_offset_fix.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/core/modules/views/src/Plugin/views/query/QueryPluginBase.php b/core/modules/views/src/Plugin/views/query/QueryPluginBase.php
```

Comment:
Why is this patch applied? Is there an issue to reference? If not then one could be created against Drupal core and patch provided there.

Reply:
Check this issue https://www.drupal.org/project/drupal/issues/3361177